controllers/suite_util_test.go

```diff
@@ -293,9 +293,9 @@ func createBuildTaskRunWithImage(resourceKey types.NamespacedName, image string)
 }
 
 func deleteAllTaskRuns() {
-	k8sClient.DeleteAllOf(ctx, &tektonapi.TaskRun{}, &client.DeleteAllOfOptions{
+	Expect(k8sClient.DeleteAllOf(ctx, &tektonapi.TaskRun{}, &client.DeleteAllOfOptions{
```

Comment:
If no objects of the kind is found, the assert is fail.

Reply:
👍  will fix in a follow-up PR